src/components/Settings/assets/video.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="20" height="12" viewBox="0 0 20 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M2 0C0.895431 0 0 0.895431 0 2V10C0 11.1046 0.89543 12 2 12H11C12.1046 12 13 11.1046 13 10V2C13 0.895431 12.1046 0 11 0H2ZM15.4855 2.7087C15.1843 2.88942 15 3.21493 15 3.56619V8.43381C15 8.78507 15.1843 9.11058 15.4855 9.2913L18.4855 11.0913C19.152 11.4912 20 11.0111 20 10.2338V1.76619C20 0.988898 19.152 0.508785 18.4855 0.9087L15.4855 2.7087Z" fill="white"/>
+</svg>
```

Comment:
please create a `JSX.component` that renders the content of this.
```
export const VideoIcon = (props) => {
  return <svg>...</svg>;
};

```
with this approach, we will be able to control the size, color, border (and more) of the vectors from within.
do not use direct SVGs like that.

Reply:
in this case, with small project - I don't need to customize colors and sizes.. are there any other advanteges to use component?